Sources/CXTestUtility/TracingSubject.swift

```diff
@@ -106,20 +103,25 @@ public class TestSubject<Output, Failure: Error>: Subject, TestLogging {
     }
 }
 
-extension TestSubject {
+// MARK: - Subscription
+
+extension TracingSubject {
     
-    public final class Inner: Subscription, CustomStringConvertible, CustomDebugStringConvertible, TestLogging {
+    public final class Subscription: CXShim.Subscription, CustomStringConvertible, CustomDebugStringConvertible {
         
-        typealias Pub = TestSubject<Output, Failure>
+        typealias Pub = TracingSubject<Output, Failure>
         typealias Sub = AnySubscriber<Output, Failure>
         
         let lock = Lock(recursive: true)
-        
-        public var name: String?
        
         var pub: Pub?
         var sub: Sub?
         
+        enum DemandState: Equatable {
```

Comment:
`DemandState` only makes sense in `TracingSubject`. I can't find a general use case so I just make it private.